data_store/controllers/notify.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from notifications_python_client.errors import HTTPError
+from notifications_python_client.notifications import NotificationsAPIClient
+from werkzeug.datastructures import FileStorage
+
+from config import Config
+from data_store.controllers.load_functions import get_or_generate_submission_id
+from data_store.db.queries import get_programme_by_id_and_round
+from submit.main.fund import FundConfig
+from submit.utils import get_friendly_fund_type
+
+
+def send_email(
+    email_address: str,
+    template_id: str,
+    notify_key: str | None,
+    personalisation: dict,
+) -> None:
+    """Send email to the specified email address via the GovUK Notify service.
+
+    :param email_address: recipient email address
+    :param template_id: template ID of GovNotify template
+    :param file: optional - file to send by email
+    :param personalisation: email personalisation values
+    :return: None
+    """
+
+    try:
+        notify_client = NotificationsAPIClient(notify_key)
+
+        notify_client.send_email_notification(
+            email_address=email_address,
+            template_id=template_id,
+            personalisation=personalisation,
+        )
+    except HTTPError as error:
+        raise error
+
+
+def send_la_confirmation_emails(
+    excel_file: FileStorage,
+    fund: FundConfig,
+    user_email: str,
+    metadata: dict,
+):
+    place_name = metadata.get("Programme Name")
+    fund_type = metadata.get("FundType_ID")
+
+    if not (place_name or fund_type):
+        raise ValueError(
```

Comment:
Raising ValueError rather that logging directly inside the function.

Reply:
I don't know if it's even possible for this `Exception` to happen and I was very tempted to delete it. But I reluctantly kept it.

Reply:
Rather than just passing in `metadata` which is a random dict, why not have this take the place_name and fund_type explicitly as strings? Then we don't have to worry about this validation.

Reply:
Changed the message.